source/_docs/configuration/packages.markdown

```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
+If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See the general documentation for [Authentication Providers](/docs/authentication/providers/#configuring-auth-providers).
+
+This is because Home Assistant processes the authentication provided early in the start-up process, even before packages are processed.
```

Comment:
Adds new-lines to ensure parsers can switch between HTML and Markdown context.

```suggestion

If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See the general documentation for [Authentication Providers](/docs/authentication/providers/#configuring-auth-providers).

This is because Home Assistant processes the authentication provided early in the start-up process, even before packages are processed.

```